examples/benchmarks/gaia.py

```diff
@@ -0,0 +1,194 @@
+        comma separated list of numbers and/or strings.
+        If you are asked for a number, don't use comma to write your number
+        neither use units such as $ or percent sign unless specified otherwise.
+        If you are asked for a string, don't use articles, neither
+        abbreviations (e.g. for cities), and write the digits in plain text
+        unless specified otherwise.
+        If you are asked for a comma separated list, apply the above rules
+        depending of whether the element to be put in the list is a number or
+        a string.
+        """.strip()
+
+sys_msg = BaseMessage.make_assistant_message(
+    role_name="Assistant",
+    content=task_prompt,
+)
+
+tools = runtime.get_tools()
+
+model = ModelFactory.create(
+    model_platform=ModelPlatformType.DEFAULT,
+    model_type=ModelType.DEFAULT,
+)
+
+
+agent = ChatAgent(
+    sys_msg,
+    model,
+    tools=tools,
+)
+
+with runtime as r:
+    r.wait()
+    result = benchmark.run(agent, "valid", level="all", subset=3)
+    print(json.dumps(result, indent=4))
+
+# ruff: noqa: E501
+"""
+Number of validation examples: 165
+Number of test examples: 300
+{
+    "total": 3,
+    "correct": 0,
+    "results": [
+        {
+            "task_id": "c61d22de-5f6c-4958-a7f6-5e9707bd3466",
+            "model_answer": "search arXiv.org for article",
+            "ground_truth": "egalitarian",
+            "tool_calls": [
+                {
+                    "func_name": "execute_code",
+                    "args": {
+                        "code": "search_arxiv('Physics and Society', '2016-08-11')"
+                    },
+                    "result": "Executed the code below:\n```py\nsearch_arxiv('Physics and Society', '2016-08-11')\n```\n> Executed Results:\nTraceback (most recent call last):\n  File \"/app/camel/camel/interpreters/internal_python_interpreter.py\", line 189, in execute\n    line_result = self._execute_ast(node)\n  File \"/app/camel/camel/interpreters/internal_python_interpreter.py\", line 253, in _execute_ast\n    return self._execute_ast(expression.value)\n  File \"/app/camel/camel/interpreters/internal_python_interpreter.py\", line 235, in _execute_ast\n    return self._execute_call(expression)\n  File \"/app/camel/camel/interpreters/internal_python_interpreter.py\", line 330, in _execute_call\n    callable_func = self._execute_ast(call.func)\n  File \"/app/camel/camel/interpreters/internal_python_interpreter.py\", line 282, in _execute_ast\n    return self._execute_name(expression)\n  File \"/app/camel/camel/interpreters/internal_python_interpreter.py\", line 366, in _execute_name\n    return self._get_value_from_state(name.id)\n  File \"/app/camel/camel/interpreters/internal_python_interpreter.py\", line 516, in _get_value_from_state\n    raise InterpreterError(f\"The variable `{key}` is not defined.\")\ncamel.interpreters.interpreter_error.InterpreterError: The variable `search_arxiv` is not defined.\n"
```

Comment:
the `"results"` would be too detailed to show here, maybe we just need to give some chore information back including 

Number of validation examples: 165
Number of test examples: 300
"total": 3,
"correct": 0,